services/modules/ipboard/views.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import unicode_literals
+
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render, redirect
+
+from authentication.decorators import members_and_blues
+from services.forms import ServicePasswordForm
+from eveonline.managers import EveManager
+
+from .manager import IPBoardManager
+from .tasks import IpboardTasks
+from .models import IpboardUser
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@login_required
+@members_and_blues()
+def activate_ipboard_forum(request):
+    logger.debug("activate_ipboard_forum called by user %s" % request.user)
+    character = EveManager.get_main_character(request.user)
+    logger.debug("Adding ipboard user for user %s with main character %s" % (request.user, character))
+    result = IPBoardManager.add_user(character.character_name, request.user.email)
+    if result[0] != "":
+        ipboard_user = IpboardUser()
```

Comment:
IMO the manager should be totally agnostic to its environment, so no database operations (mumble is an exception). I did consider moving the `add_user` database operations to the Tasks class, but I just decided to leave the existing behaviour intact for now. Mainly because I didn't want to futz around with managing passing different error states through to the view. Most (all currently) of them will only have a single area responsible for user creation.